config-default.yml

```
@@ -269,7 +269,7 @@ filter:
     notify_user_domains:     false
 
     # Filter configuration
-    ping_everyone:             true  # Ping @everyone when we send a mod-alert?
+    ping_moderators:             true  # Ping @everyone when we send a mod-alert?
```

The comment does not reflect the actual value; the comment still mentions `@everyone`.

```suggestion
    ping_moderators:             true  # Ping @Moderators when we send a mod-alert?
```

Or we could simply remove this unnecessary comment to prevent these kind of "left behind" comments during refactor

```suggestion
    ping_moderators:             true
```